old/production-configs/fr.openfisca.org/config/fr.openfisca.org.conf

```diff
@@ -0,0 +1,26 @@
+server {
```

Comment:
I'm a little confused by this one. I understand the idea of storing in `old` the config of old services, but this seems to be an old version of a current service, which doesn't seem super useful.

Should we just remove it?